code/analysis/DataTable.csv

```diff
@@ -0,0 +1,675 @@
+Package,	,	,	,	,	,Class Instances
```

Comment:
I don't think this `DataTable.csv` should be part of this PR.

Reply:
Oh sorry, I didn't know it added that. I was looking at the data table from the other issues so I might've done something to it there.